ballerina/tests/sms-sandbox-tests.bal

```diff
@@ -0,0 +1,153 @@
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412344502");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412344503", ES_419);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678903");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678904", ES_ES);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678904");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678905", DE_DE);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678905");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678906", FR_FR);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678906");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678907", FR_CA);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678907");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678908", IT_IT);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678908");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678909", JA_JP);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678909");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678910", PT_BR);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678910");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678911", KR_KR);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678911");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678912", ZH_CN);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678912");
+
+    check amazonSNSClient->createSMSSandboxPhoneNumber("+9412345678913", ZH_TW);
+    runtime:sleep(2);
+    check amazonSNSClient->deleteSMSSandboxPhoneNumber("+9412345678913");
+}
+
+@test:Config {
+    groups: ["sms-sandbox"]
+}
+function listSMSSandboxPhoneNumbersTest() returns error? {
+    stream<SMSSandboxPhoneNumber, Error?> phoneNumberStream = amazonSNSClient->listSMSSandboxPhoneNumbers();
+    SMSSandboxPhoneNumber[] phoneNumbers = check from SMSSandboxPhoneNumber phoneNumber in phoneNumberStream
+        select phoneNumber;
+
+    test:assertEquals(phoneNumbers.length(), 10);
+    _ = from SMSSandboxPhoneNumber phoneNumber in phoneNumbers
+        do {
+        test:assertTrue(phoneNumber.phoneNumber.startsWith("+947719"));
+        test:assertTrue(phoneNumber.status == PENDING || phoneNumber.status == VERIFIED);
+        };
+}
+
+@test:Config {
+    groups: ["sms-sandboxx"]
+}
+function getSMSSanboxAccountStatusTest() returns error? {
+    boolean status = check amazonSNSClient->getSMSSandboxAccountStatus();
+    test:assertTrue(status);
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in [020b0a4](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/40/commits/020b0a40d31509c3e7b499cf2ea5c620bbfc9b2b)